stdlib/email/quoprimime.pyi

```diff
@@ -0,0 +1,13 @@
+def header_check(octet: int) -> bool: ...
+def body_check(octet: int) -> bool: ...
+def header_length(bytearray: bytes) -> int: ...
+def body_length(bytearray: bytes) -> int: ...
+def unquote(s: str) -> str: ...
```

Comment:
Not sure if this is useful, but it will work at runtime:

```suggestion
def unquote(s: str | bytes) -> str: ...
```

Reply:
yeah let's do that then as it makes unqoute and quote consistent 👍